modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -9276,6 +9291,19 @@ type TargetEnvironment {
     observationTime: Timestamp!
   ): [GuideEnvironment!]!
 
+  """
+  Availability of guide stars during a specified time range.
+  There can be multiple `GuideAvailabilityPeriod`s returned if availability changes over the time
+  range. In this case, the `end` of one period will be the same as the `start` of the next period.
+  """
+  guideAvailability(
```

Comment:
I added this to the `TargetEnvironment` for lack of a better location. It can be moved if someone has a better suggestion.